install/test/fixtures.go

```diff
@@ -419,20 +421,21 @@ static_resources:
                   cluster: admin_port_cluster
                   prefix_rewrite: /stats/prometheus
           stat_prefix: prometheus
-        name: envoy.http_connection_manager
+          tracing:
+            provider:
+              typed_config:
+                '@type': type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
+                collector_cluster: zipkin
+                collector_endpoint: /api/v1/spans
```

Comment:
i think this should be v2
```suggestion
                collector_endpoint: /api/v2/spans
```